transport_type: 0

ethernet_server_info: "192.168.0.10"

# debug enable
debug_enable: false

# hdr settings
hdr_mode : 2
quality_over_exposed : 500
quality_over_exposed_serious : 800
quality_weak : 120
quality_weak_serious : 80
simple_hdr_max_integration : 2000
simple_hdr_min_integration : 200
super_hdr_frame_number : 3
super_hdr_max_integration : 20000
hdr_disable_integration_time : 1000

# kalman filter
filter_type : 0
kalman_K : 0.3
kalman_threshold : 200
num_check : 2

# smooth filter
smooth_filter_type : 0
smooth_filter_arg1 : 0

# mounting angle
mounting_angle_enable : false
mounting_angle_angle_vertical : 0
mounting_angle_height : 0

# edge detection enable
edge_detection_enable : true
edge_detection_threshold_value : 1000
